Dockerfile

```diff
@@ -12,11 +12,12 @@ RUN apt update -y && apt install python3-pip git -y && pip3 install --no-cache-d
 
 ADD Pipfile Pipfile.lock /httpbin/
 WORKDIR /httpbin
-RUN /bin/bash -c "pip3 install --no-cache-dir -r <(pipenv lock -r)"
+RUN /bin/bash -c "pip3 install --no-cache-dir -r <(pipenv requirements)"
 
 ADD . /httpbin
 RUN pip3 install --no-cache-dir /httpbin
+RUN chmod +x /httpbin/httpbin.bash
 
 EXPOSE 80
 
-CMD ["gunicorn", "-b", "0.0.0.0:80", "httpbin:app", "-k", "gevent"]
+ENTRYPOINT ["/httpbin/httpbin.bash"]
```

Comment:
I'm in favor of this change, with one nitpick:

Any reason to prefer `ENTRYPOINT` to `CMD` here?  I personally prefer `CMD` since it's more common and easier to override.

Reply:
I'd say my choice of `ENTRYPOINT` also comes down to personal preference. I prefer it for the exact same reason as you prefer `CMD`: It is more cumbersome to override. My thought behind this is that it is generally not needed to override, and using `ENTRYPOINT` avoids accidents. As both have their trade-offs, I'm fine with either. I'll push the update in a moment.